crates/rooch/src/commands/init.rs

```diff
@@ -5,27 +5,37 @@ use crate::cli_types::{CommandAction, WalletContextOptions};
 use crate::utils::read_line;
 use async_trait::async_trait;
 use clap::Parser;
+use regex::Regex;
 use rooch_config::store_config::StoreConfig;
 use rooch_config::{rooch_config_dir, Config, ROOCH_CLIENT_CONFIG, ROOCH_KEYSTORE_FILENAME};
 use rooch_key::keystore::{AccountKeystore, FileBasedKeystore, Keystore};
 use rooch_rpc_client::client_config::{ClientConfig, Env};
+use rooch_types::error::RoochError;
 use rooch_types::{crypto::BuiltinScheme, error::RoochResult};
 use std::fs;
 
 /// Tool for init with rooch
 #[derive(Parser)]
 pub struct Init {
-    /// Accept defaults config, default true
-    #[clap(short = 'y', long = "yes", default_value_t = true)]
+    /// Accept defaults config
+    #[clap(short = 'd', long = "default")]
```

Comment:
Seem like the `accept_defaults` is useless. I think we can remove it. If the users want to use a custom value, they can edit the config file later. @wow-sven

Reply:
Removed. @jolestar